.github/workflows/test.yml

```diff
@@ -56,15 +55,22 @@ jobs:
     - name: Install package dependencies
       run: mix deps.get
 
+    - name: Remove compiled application files
+      run: mix clean
+
+    - name: Compile dependencies
+      run: mix compile
+
     - name: Check documentation quality locally
       run: mix inch
 
     - name: Report documentation quality
+      if: github.event_name == 'push'
```

Comment:
Would you please add context for this change.

Reply:
Well, you already described the issue https://github.com/duffelhq/paginator/pull/105 with multiple runs of this check in case of pull request, but the solution wasn't the best.
I've returned back the part with running CI on pull requests (see the top of the document), but disabled this step of sending the report in such the case. It will be executed only on pushing some code to the repository and will be skipped in case of creating a pull request.

Reply:
Thank you for articulating the reason for this change!